packages/features/bookings/lib/getWebhookPayloadForBooking.ts

```diff
@@ -0,0 +1,37 @@
+import type { EventTypeInfo } from "@calcom/features/webhooks/lib/sendPayload";
+import type { CalendarEvent } from "@calcom/types/Calendar";
+
+export const getWebhookPayloadForBooking = ({
```

Comment:
Webhook payload for booking should remain common across different triggers. But right now it's different in small ways due to it being not DRY.